CHANGES.rst

```diff
@@ -268,9 +212,42 @@ Contributor-facing changes
 
 
 
+Removals and backward incompatible breaking changes
+---------------------------------------------------
+
+- The shutdown logic in 3.9 waited on all tasks, which caused issues with some libraries.
+  In 3.10 we've changed this logic to only wait on request handlers. This means that it's
+  important for developers to correctly handle the lifecycle of background tasks using a
+  library such as ``aiojobs``. If an application is using ``handler_cancellation=True`` then
+  it is also a good idea to ensure that any :func:`asyncio.shield` calls are replaced with
+  :func:`aiojobs.aiohttp.shield`.
+
+  Please read the updated documentation on these points:
+  https://docs.aiohttp.org/en/stable/web_advanced.html#graceful-shutdown
+  https://docs.aiohttp.org/en/stable/web_advanced.html#web-handler-cancellation
+
+  -- by :user:`Dreamsorcerer`
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8495`.
+
+
+
+
 Miscellaneous internal changes
 ------------------------------
 
+- Improve performance of filtering cookies -- by :user:`bdraco`.
+
+  This change is a followup to the improvements in :issue:`7583`
```

Comment:
missing trailing period?

Reply:
Deleted since 8535 is now a symlink to 7583